### PR TITLE
Update JenkinsfileBase to use TKG iterations

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Review by smlambert — APPROVED** [review]
Thanks @renfeiw - for this and its related PR https://github.com/adoptium/TKG/pull/356 !

I imagine we should kick off a few Grinders where PARALLEL = NodesByIterations to ensure it behaves as expected.

**Comment by renfeiw** [comment]
> Thanks @renfeiw - for this and its related PR [adoptium/TKG#356](https://github.com/adoptium/TKG/pull/356) !
> 
> I imagine we should kick off a few Grinders where PARALLEL = NodesByIterations to ensure it behaves as expected.

Thank you @smlambert Yes, it passed the grinder with parallel. Actually we will hold off this change as stated in https://github.com/adoptium/TKG/pull/356#issuecomment-1256409858. We will merge the change in TKG first, and once the feature is complete and then we merge this PR.

**Comment by llxia** [comment]
Close this as obsolete

**Pull request closed without merging**
